website/docs/guides/middlewares/middlewares.md

```diff
@@ -17,6 +17,12 @@ Every product/consumer has its own Middlewares instances, so the instances are n
 
 Middlewares are instantiated by the configured dependency injection container, so every dependency configured by your container can be delivered in the middleware constructor.
 
+:::info
+
+Use overloads of the `Add<TMiddleware>(MiddlewareLifetime)` method when registering middlewares with your DI container to control middleware instance lifetime.
```

Comment:
Hi @mhkolk,

Thank you for improving the documentation. 

Would you mind creating a section specific for this information in this page? Maybe naming it like "Configure Middleware Lifetime", similar to the one on [TypedHandlers](https://farfetch.github.io/kafkaflow/docs/guides/middlewares/typed-handler-middleware#configuring-handler-lifetime)

There is more useful information that we can share here:
- The available lifetimes ([here](https://github.com/Farfetch/kafkaflow/blob/master/src/KafkaFlow.Abstractions/MiddlewareLifetime.cs))
- The default lifetime (ConsumerOrProducer)
- Code sample on how to configure the middleware lifetime

Reply:
This has been opened for a while and I have to say I don't have the time atm to expand what I already added.

Reply:
No problem @mhkolk, we will merge as-is, and then we can open a new PR with the changes proposed.

Thank you for your contribution.